bee-message/src/output/byte_cost.rs

```diff
@@ -0,0 +1,106 @@
+// Copyright 2021-2022 IOTA Stiftung
```

Comment:
```suggestion
// Copyright 2022 IOTA Stiftung
```

Reply:
I started working on it last year.

Reply:
My assumption was that the copyright date is related to when it actually makes it to the repository but this is all very obscure to me, feel free ti ignore